lib/signaling/v2/room.js

```diff
@@ -190,14 +200,14 @@ class RoomV2 extends RoomSignaling {
   /**
    * @private
    */
-  _getOrCreateTrackReceiverDeferred(id) {
+  _getOrCreateTrackReceiverDeferred(id, idOrMid = 'id') {
```

Comment:
NIT: now that id could be mid, could we rename this variable to propertyValue 

```suggestion
  _getOrCreateTrackReceiverDeferred(propertyValue, propertyName = 'id') {
```

Reply:
I think the current argument names are more descriptive.